kratos/containers/geometry_container.h

```diff
@@ -87,6 +87,13 @@ class GeometryContainer
         : mGeometries(NewGeometries)
     {}
 
+    /// Components const Constructor
+    GeometryContainer(
+        GeometriesMapType const& NewGeometries)
+        : mGeometries(NewGeometries)
+    {}

```

Comment:
Please someone else from @KratosMultiphysics/technical-committee confirm this is fine. 

It is needed to call the const version of the `WriteGeometries` from the `model_part_io.h`